Sources/Persistence/CoreData/CoreDataEntity.swift

```diff
@@ -1,6 +1,6 @@
 import CoreData
 
-public protocol CoreDataEntity: class, NSFetchRequestResult {
+public protocol CoreDataEntity: AnyObject & NSFetchRequestResult {
```

Comment:
Do we still need to specify `AnyObject` here? As for something to conform to `NSFetchRequestResult`, it must also conform to `NSObjectProtocol`, which is a class protocol.

Reply:
You're absolutely right! Will remove redundant `AnyObject` conformance 😉